paddle/fluid/inference/analysis/argument.h

```diff
@@ -169,6 +169,9 @@ struct Argument {
 
   // Scales for variables to be quantized
   DECL_ARGUMENT_FIELD(quant_var_scales, QuantVarScales, VarQuantScale);
+
+  // Destination data layout of quantization
+  DECL_ARGUMENT_FIELD(quantization_data_layout, DataLayout, std::string);
```

Comment:
You do not have to declare any field here for an optional pass attribute.

Reply:
Alrighty, didn't know that. Will remove the attribute.